### ISSUE TITLE
Partition field not showing up in generated table 

### ISSUE THREAD

**Issue by pedrosdf** [firebase/extensions#1577]
Issue summary:

The customer currently has installed the Stream Firestore to BigQuery extension in Firebase. 

Using the Stream Firestore to BigQuery extension. After configuring the extension, the field used for partition (timestamp-field), the field does not exist in the generated table.

Expected behavior: The field used for partition, must be shown as a column in both tables generated.

Current screen (customer project):
![image (4)](https://github.com/firebase/extensions/assets/90926921/bbd323e9-9d62-4863-9746-276b37da2dd8)

Expected screen (this is other project):
![image (5)](https://github.com/firebase/extensions/assets/90926921/648cd8e5-005a-4b87-8b04-d06eb07ccf0c)

Current configuration:

Cloud Functions location: <location>
BigQuery Dataset location: <location>
Project Id: <cx-project>
Collection path: <node>
Enable Wildcard Column field with Parent Firestore Document IDs (Optional): false
Dataset ID: <dataset-id>
Table ID: <table>
BigQuery SQL table Time Partitioning option type (Optional): MONTH
BigQuery Time Partitioning column name (Optional): <timestamp-field>
Firestore Document field name for BigQuery SQL Time Partitioning field option (Optional): <timestamp-field>
BigQuery SQL Time Partitioning table schema field(column) type (Optional): TIMESTAMP
BigQuery SQL table clustering (Optional): Parameter not set
Backup Collection Name (Optional): Parameter not set
Transform function URL (Optional): Parameter not set
Use new query syntax for snapshots: no

Business Impact: Customer lost visibility of the field in the generated BigQuery tables.

* Why is the unexpected behavior happening?
* Is there any change that should be made in the settings?

Note: The customer already opened a ticket on Google Cloud Support, but they recommended opening the issue here in GitHub, since the issue comes from a Firebase extension and not directly from the Firebase product.

[1]: https://extensions.dev/extensions/firebase/firestore-bigquery-export

**Comment by dackers86** [comment]
Hi @pedrosdf 

Partitioning cannot be added to already existing table. Did this table already exist when setting up the extension?

**Comment by seancarroll** [comment]
I'm noticing the same thing as well using version `0.1.29`. I'm setting `TIME_PARTITIONING_FIRESTORE_FIELD` to a top level field but field isn't created in the generated table. I've looked through the fsexportbigquery function logs but not seeing any logs that point to a specific problem. I do see "No valid table reference is available. Skipping partitioning". The logs i get are essentially

```
BigQuery dataset already exists:
Creating BigQuery table: 
No valid table reference is available. Skipping partitioning
Cannot partition an existing table
```

**Comment by pr-Mais** [comment]
This issue has been fixed in #1794, closing.
